space-wizards/space-station-14
# Add "Syndicate Puddle-Jumper" blueprint for admins to spawn in

## About the PR
Serves as a way to let admins cause nukies.

Admin usage guide (and testing procedure):

1. Ensure 0, 0 is clear of obstructions
2. `loadbp 1 Maps/syndipuddle.yml`
3. teleport nukies to 0, 0 (where it will have shown up)
4. Supply uplinks if applicable
5. Make sure to forcibly end the round when the nuke goes off

Specific shuttle features:

+ Has a "technically feasible to win" roundstart kit - arguably ready-to-use uplinks-or-not
+ Has a tiny atmos setup to recover from airlock accidents - first person to work out the intended use-case of the one-way injection port and valve setup gets a metaphorical biscuit
+ Has power and gravity

Nothing worth changelogging here

**Screenshots**
![image](https://user-images.githubusercontent.com/22304167/141355059-4f2e1ba5-7d57-42dd-a037-6735add2f378.png)
![image](https://user-images.githubusercontent.com/22304167/141355095-34d8902c-72a5-43c0-b43e-928fbf6ace3e.png)
